velox/functions/sparksql/Arithmetic.h

```diff
@@ -21,7 +21,10 @@
 #include <system_error>
 #include <type_traits>
 
+#include "velox/common/base/BitUtil.h"
```

Comment:
This header is not needed, is it? Same for StringImpl.h.

Reply:
sorry for that, I guess FB lint failed due to this, already removed.